lib/bundles/inspec-supermarket/api.rb

```diff
@@ -11,11 +11,11 @@ class API
 
     # displays a list of profiles
     def self.profiles(supermarket_url = SUPERMARKET_URL)
-      url = "#{supermarket_url}/api/v1/tools"
-      _success, data = get(url, { start: 0, items: 100, order: 'recently_added' })
+      url = "#{supermarket_url}/api/v1/tools-search"
+      _success, data = get(url, { type: 'compliance_profile', items: 100, order: 'recently_added' })
```

Comment:
_Need_? Maybe not. Depends on which makes sense as default: alphabetical by `name` or reverse chronological by creation with `recently_added`. Name might make more sense.

Sadly, v1 of the Supermarket API does not include next or previous references (a la  HATEOAS) in the response, so seeing more than the first 100 of either set of results is left as an exercise to the client.